### ISSUE TITLE
Uploading with Shiny to remote workspace fails

### ISSUE THREAD

**Issue by Hong-Revo** [Microsoft/RTVS#3283]
Most of the examples from `shiny::runExample` work fine when run from a remote workspace, except for `"09_upload"`.
```
library(shiny)
runExample("09_upload")

Listening on http://127.0.0.1:4828
HTTP server for remote GPU DSVM starting ...
HTTP server for remote GPU DSVM listening on http://127.0.0.1:60694. 
HTTP server for remote transfers data from remote R machine.
Server at DESKTOP-TBHQGUH http://127.0.0.1:60694 connects to GPU DSVM https://xxx-xxx.cloudapp.azure.com:5444/.
HTTP server for remote stopped with following exception:
You must provide a request body if you set ContentLength>0 or SendChunked==true.  Do this by calling [Begin]GetRequestStream before [Begin]GetResponse.
HTTP server for remote GPU DSVM stopped.
HTTP server for remote stopped with following exception:
An internal WebSocket error occurred. Please see the innerException, if present, for more details. 
```

The error occurs once you select a file to upload.

**Comment by MikhailArkhipov** [comment]
@karthiknadig - what's the cause?

**Comment by karthiknadig** [comment]
There seems to be a issue with the way we handle files when submitted via form input. I figured out the client side problem. Server side seems have a bug as well. I have not investigated server side yet.

Client side issue was, we were looking for a seek-able network stream to get the length of the input data. When a file is being uploaded, the stream is read-only and not seek-able. we ended up skipping the upload. That leads to the error that you see above. However, fixing that on client side is not enough, since we need to pass it all the way to RHost, there is one more change that is needed on the broker side.
